examples/accounts.py

```diff
@@ -0,0 +1,129 @@
+#list account info., incl. past transactions
+parser_create = subparsers.add_parser('list', description = 'individual account information display')
+parser_create.add_argument('account', help = 'specify account')                         
+#transfer money between accounts
+parser_create = subparsers.add_parser('transfer', description = 'transer funds between accounts')
+parser_create.add_argument('transferAmount', help ='quantity of money to transfer', type=decimal.Decimal, default = decimal.Decimal('0.00'))
+parser_create.add_argument('pastLocation', help ='account from which funds are withdrawn')
+parser_create.add_argument('futureLocation', help = 'account to which funds are deposited') 
+parser_create.add_argument('memo', help = 'explanation of transfer', nargs = argparse.REMAINDER)
+#withdraw money from account (check)
+parser_create = subparsers.add_parser('check', description = 'withdraw money via check')
+parser_create.add_argument('checkNumber', help = 'number of check')
+parser_create.add_argument('account', help = 'account from which to withdraw money')
+parser_create.add_argument('amount', help = 'amount withdrawn', type = decimal.Decimal, default = decimal.Decimal('0.00'))
+parser_create.add_argument('memo', help = 'check memo', nargs = argparse.REMAINDER)
+#deposit money into account
+parser_create = subparsers.add_parser('deposit', description = 'deposit money into account')
+parser_create.add_argument('account', help = 'account in which to deposit money')
+parser_create.add_argument('amount', help = 'total deposit', type = decimal.Decimal, default = decimal.Decimal('0.00'))
+parser_create.add_argument('memo', help = 'source of deposit', nargs = argparse.REMAINDER)
+#withdraw money from account (cash)
+parser_create = subparsers.add_parser('withdraw', description = 'withdraw amount of cash from account')
+parser_create.add_argument('account', help = 'account from which to withdraw money')
+parser_create.add_argument('amount', help = 'total withdrawl', type = decimal.Decimal, default = decimal.Decimal('0.00'))
+parser_create.add_argument('memo', help = 'reason for withdrawl', nargs = argparse.REMAINDER)
+args = parser.parse_args()
+
+#check to see if args.date is a string -- convert to datetime object if so
+if isinstance(args.date, str) == True:
+    args.date = datetime.datetime.strptime(args.date, '%Y-%m-%d')
+    
+with PersistentObjectPool(args.filename, flag='c') as pop:
+    if pop.root is None:
+        pop.root = pop.new(PersistentDict)
+    accounts = pop.root
+    if args.subcommand == 'create':
+        accounts[args.account] = [[args.date, decimal.Decimal(args.amount), 'Initial account balance']]
+        print("Created account '" + args.account + "'.")                       
+    elif args.subcommand == 'list':
+        L1 = ("Date         Amount  Balance  Memo\n"
+              "----------  -------  -------  ----")
+        print(L1)
```

Comment:
There's nothing wrong with what you have here.  However, it is a general principle in python that in most places where you could use a variable name you can instead use an expression.  This applies particularly to function arguments.  You have "L1 = <expression>" followed by "print(L1)".  You could instead write "print(<expression>)".  That is, the above two lines would become:

    print("Date         Amount  Balance  Memo\n"
             "----------  -------  -------  ----")

This is how you will typically see print statements written in Python programs.